packages/library-sync/src/commonMain/kotlin/io/realm/kotlin/mongodb/App.kt

```diff
@@ -101,6 +102,14 @@ public interface App {
      */
     public suspend fun login(credentials: Credentials): User
 
+    /**
+     * Create a [Flow] of [AuthenticationChange]-events to receive notifications of updates to all
+     * app user authentication states, like login and logout.
```

Comment:
```suggestion
     * app user authentication states: login, logout and removal.
```

Currently there are only three states, we could mention them all.